### DISCUSSION TITLE
API documentation menu not working

### DISCUSSION THREAD

**Discussion by petersmythe** [Investec-Developer-Community/.github#25] [Bugs]
I can't recall if anyone has documented this problem before but the left side menu does not work on API documentation pages e.g. 

https://developer.investec.com/za/api-products/documentation/SA_PB_Account_Information#operation/transferv2

![image](https://github.com/user-attachments/assets/94d91fcc-fb6f-4d33-a2f1-bcac9f3b5a4f)

does not bring transferv2 into view.

**Comment by neilwithers** [comment]
Hi Peter, thanks for letting us know. 

This is on our backlog and we will let you know when it's fixed.

**Comment by davidfraser** [comment]
Not sure if this just hasn't rolled out yet, but it still doesn't work for me using Google Chrome. Clicking under _Account information_ on _Get Profiles_ changes the URL in the URL bar to https://developer.investec.com/za/api-products/documentation/SA_PB_Account_Information#operation/profiles but the page doesn't update to that location, although it does show the correct page on account info.

**Reply by programmable-banking-community** [reply]
Hi @davidfraser. Thanks for checking! It is a ReDoc issue that we've been trying to find a workaround for. We're currently looking at some other approaches to our formal docs to make it more dev-friendly for you. I will let you know as soon as we have something to test - I would love your feedback on it. :) Thanks.
